DayNightQ10.Rmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "Daynight-Q10"
+author: "Colin Wu"
+date: "`r Sys.Date()`"
+output: 
+  html_document:
+    code_folding: hide
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(cosore)
+library(dplyr)
+db_info <- csr_database()
+results <- list()
+```
+```{r hasRh}
+hasRh <- function(x, msmt_col, name_col){
+   dt_msmt <- x[msmt_col]
+   dt_name <- x[name_col]
+   if(grepl("Rh", dt_msmt)){
+      print(toString(dt_name))
+      return(dt_name)
+   }
+   return(NULL)
+}
+```
+
+## Goal 1
+```{r goal1, eval=TRUE, echo=FALSE}
+rh_dataset_names <- db_info[grep("Rh", db_info$CSR_MSMT_VAR),1][[1]]
+for(rhds in rh_dataset_names){
+  print(rhds)
+}
+```
+```{r new-goal1}
+rh_dataset_names <- filter(db_info, grepl("Rh", CSR_MSMT_VAR))
+rh_dataset_names <- pull(rh_dataset_names, CSR_DATASET)
+print(rh_dataset_names)
+```
+
+## Goal 2
+```{r Goal 2}
+results <- list()
+for(rhds in rh_dataset_names){
+  ds_data <- csr_dataset(rhds)
+  ports_table <- ds_data$ports
+  apply(ports_table, 1, hasRh, 2, 1)
```

Comment:
Hmmm...what's the `apply` doing here? Why is it needed?

Reply:
The apply is only really there to print out the port numbers. I'll remove it